--- vlc-android/compile-libvlc.sh.omv~	2018-01-30 22:31:02.354593081 +0100
+++ vlc-android/compile-libvlc.sh	2018-01-30 22:32:47.448753785 +0100
@@ -283,19 +283,9 @@ VLC_CONTRIB="$VLC_SRC_DIR/contrib/$TARGE
 # try to detect NDK version
 REL=$(grep -o '^Pkg.Revision.*[0-9]*.*' $ANDROID_NDK/source.properties |cut -d " " -f 3 | cut -d "." -f 1)
 
-# NDK 15 and after drops support for old android platforms (bellow
-# ANDROID_API=14) but these platforms are still supported by VLC 3.0.
-# TODO: Switch to NDK 15 when we drop support for old android plaftorms (for VLC 4.0)
-if [ "$REL" -eq 14 ]; then
-    if [ "${HAVE_64}" = 1 ];then
-        ANDROID_API=21
-    else
-        ANDROID_API=9
-    fi
-else
-    echo "NDK v14 needed, cf. https://developer.android.com/ndk/downloads/older_releases.html#ndk-14-downloads"
-    exit 1
-fi
+# We don't care about prehistoric Android -- FMO is based on
+# API 27, so let's use that
+ANDROID_API=27
 
 NDK_FORCE_ARG=
 NDK_TOOLCHAIN_DIR=${PWD}/toolchains/${PLATFORM_SHORT_ARCH}
