--- vlc-android/vlc/contrib/src/protobuf/rules.mak.0004~	2018-01-31 21:47:17.973758644 +0100
+++ vlc-android/vlc/contrib/src/protobuf/rules.mak	2018-01-31 21:59:08.254430738 +0100
@@ -1,5 +1,5 @@
 # protobuf
-PROTOBUF_VERSION := 3.1.0
+PROTOBUF_VERSION := 3.5.1
 PROTOBUF_URL := https://github.com/google/protobuf/releases/download/v$(PROTOBUF_VERSION)/protobuf-cpp-$(PROTOBUF_VERSION).tar.gz
 
 PKGS += protobuf
@@ -16,7 +16,7 @@ DEPS_protobuf = zlib $(DEPS_zlib)
 
 protobuf: protobuf-$(PROTOBUF_VERSION)-cpp.tar.gz .sum-protobuf
 	$(UNPACK)
-	mv protobuf-3.1.0 protobuf-3.1.0-cpp
+	mv protobuf-$(PROTOBUF_VERSION) protobuf-$(PROTOBUF_VERSION)-cpp
 	$(APPLY) $(SRC)/protobuf/dont-build-protoc.patch
 	$(MOVE)
 
--- vlc-android/vlc/contrib/src/protobuf/SHA512SUMS.0004~	2018-01-31 21:47:17.973758644 +0100
+++ vlc-android/vlc/contrib/src/protobuf/SHA512SUMS	2018-01-31 21:47:19.497760120 +0100
@@ -1 +1 @@
-9f85a98e55cbc9f245a3079d5a597f778454bc945f0942cb10fbdfbde5fe12b17d6dda93d6a8d5281459ad30a3840be7e0712feb33a824226884e7e4da54a061  protobuf-3.1.0-cpp.tar.gz
+195ccb210229e0a1080dcdb0a1d87b2e421ad55f6b036c56db3183bd50a942c75b4cc84e6af8a10ad88022a247781a06f609a145a461dfbb8f04051b7dd714b3  protobuf-3.5.1-cpp.tar.gz
--- vlc-android/vlc/extras/tools/packages.mak.0004~	2018-01-31 21:47:17.986758656 +0100
+++ vlc-android/vlc/extras/tools/packages.mak	2018-01-31 21:47:19.497760120 +0100
@@ -42,7 +42,7 @@ SED_URL=$(GNU)/sed/sed-$(SED_VERSION).ta
 ANT_VERSION=1.9.7
 ANT_URL=$(APACHE)/ant/binaries/apache-ant-$(ANT_VERSION)-bin.tar.bz2
 
-PROTOBUF_VERSION := 3.1.0
+PROTOBUF_VERSION := 3.5.1
 PROTOBUF_URL := https://github.com/google/protobuf/releases/download/v$(PROTOBUF_VERSION)/protobuf-cpp-$(PROTOBUF_VERSION).tar.gz
 
 BISON_VERSION=3.0.4
