https://github.com/Aptoide/aptoide-client-v8/issues/1478

diff --git a/app/build.gradle b/app/build.gradle
index 89f6c7b090..487574dab6 100644
--- a/app/build.gradle
+++ b/app/build.gradle
@@ -482,10 +482,6 @@ dependencies {
 
   kapt "com.google.dagger:dagger-compiler:${DAGGER_VERSION}"
 
-  implementation("com.asfoundation:appcoins-contract-proxy:${APPC_CONTRACT_PROXY_VERSION}") {
-    exclude group: 'org.web3j'
-  }
-
   //careful updating this because of support library conflicts
   implementation "com.airbnb.android:lottie:${LOTTIE_VERSION}"
 
diff --git a/app/src/main/java/cm/aptoide/pt/app/view/donations/utils/GenericPaymentIntentBuilder.java b/app/src/main/java/cm/aptoide/pt/app/view/donations/utils/GenericPaymentIntentBuilder.java
index 966a2c0dc3..140f24704d 100644
--- a/app/src/main/java/cm/aptoide/pt/app/view/donations/utils/GenericPaymentIntentBuilder.java
+++ b/app/src/main/java/cm/aptoide/pt/app/view/donations/utils/GenericPaymentIntentBuilder.java
@@ -4,8 +4,6 @@ import android.app.PendingIntent;
 import android.content.Context;
 import android.content.Intent;
 import android.net.Uri;
-import com.asf.appcoins.sdk.contractproxy.AppCoinsAddressProxyBuilder;
-import com.asf.appcoins.sdk.contractproxy.AppCoinsAddressProxySdk;
 import com.google.android.gms.common.util.Hex;
 import com.google.gson.Gson;
 import io.reactivex.Single;
@@ -43,13 +41,10 @@ public class GenericPaymentIntentBuilder {
   public static PendingIntent buildBuyIntent(Context context, String skuId, String value,
       String walletAddress, String packageName, String transferType, String payload,
       boolean debug) {
-    AppCoinsAddressProxySdk proxySdk = new AppCoinsAddressProxyBuilder().createAddressProxySdk();
     int networkId = debug ? ROPSTEN_NETWORK_ID : MAIN_NETWORK_ID;
 
-    Single<String> getTokenContractAddress = proxySdk.getAppCoinsAddress(networkId)
-        .subscribeOn(Schedulers.io());
-    Single<String> getIabContractAddress = proxySdk.getIabAddress(networkId)
-        .subscribeOn(Schedulers.io());
+    Single<String> getTokenContractAddress = Single.just("");
+    Single<String> getIabContractAddress = Single.just("");
 
     return Single.zip(getTokenContractAddress, getIabContractAddress,
         (tokenContractAddress, iabContractAddress) -> buildPaymentIntent(context, networkId, skuId,
@@ -124,4 +119,4 @@ public class GenericPaymentIntentBuilder {
       this.payload = payload;
     }
   }
-}
\ No newline at end of file
+}
diff --git a/build.gradle b/build.gradle
index 89d33ff6a1..e6b6e06ef9 100644
--- a/build.gradle
+++ b/build.gradle
@@ -21,7 +21,6 @@ allprojects {
     jcenter()
     maven { url 'https://oss.sonatype.org/content/repositories/snapshots' }
     maven { url "https://www.jitpack.io" }
-    maven { url "https://dl.bintray.com/asf/asf" }
 
     google()
     maven { url 'https://maven.google.com' }
